### PR TITLE
Appoint a rep claimant screens

### PR DESCRIPTION
**Pull request by cosu419**
## Summary
Appoint a Rep
- Added claimant relationship screen
- Added confirm claimant personal information screen

Find a Rep
- Copy changes, typos
- Conditional rendering for Close button

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/85623
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86026
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86184
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/86190
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/67741
- https://github.com/department-of-veterans-affairs/va.gov-team/issues/66104

## Screenshots
image

### CODE REVIEW

File: src/applications/representative-appoint/prefill-transformer.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { prefillBankInformation } from 'platform/forms-system/src/js/definitions/directDeposit';
```
Comment (holdenhinkle) [inline src/applications/representative-appoint/prefill-transformer.js:1]: Are you going to clean this file up later with the stuff we don't need? I'm noticing bank information which is not part of the form.

Comment (holdenhinkle) [reply src/applications/representative-appoint/prefill-transformer.js:1]: Same question applies to the schema and fixtures files.

**Review by holdenhinkle — DISMISSED** [review]
LGTM. Just 1 comment and a couple of questions (which I think I know the answer to).

**Review by holdenhinkle — DISMISSED** [review]
LGTM

**Review by holdenhinkle — APPROVED** [review]

**Review by IGallupSoCo — APPROVED** [review]
LGTM

**Pull request merged**